index.html

```diff
@@ -5,180 +5,183 @@
-                  <li class="html-css">HTML/CSS</li>
-                  <li class="ruby">Ruby on Rails</li>
-                  <li class="javascript">JavaScript</li>
-               </ul>
-               <button type="button"><span>See this project</span><span><img class="arrow-right" src="images/Union.svg" alt="arrow right"></span></button>
-              </div>
-            </div>
-  
-            <div class="project-division">
-              <img src="images/Laptop-Left.png" alt="laptops">
-              <div class="project-name">
-                <h4>Project name goes here</h4>
-                <ul class="language-info">
-                  <li class="html-css">HTML/CSS</li>
-                  <li class="ruby">Ruby on Rails</li>
-                  <li class="javascript">JavaScript</li>
-               </ul>
-               <button type="button"><span>See this project</span><span><img class="arrow-right" src="images/Union.svg" alt="arrow right"></span></button>
-              </div>
-            </div>
-  
-            <div class="project-division">
-              <img src="images/Laptop-Pink.png" alt="laptops">
-              <div class="project-name">
-                <h4>Project name goes here</h4>
-                <ul class="language-info">
-                  <li class="html-css">HTML/CSS</li>
-                  <li class="ruby">Ruby on Rails</li>
-                  <li class="javascript">JavaScript</li>
-               </ul>
-               <button type="button"><span>See this project</span><span><img class="arrow-right" src="images/Union.svg" alt="arrow right"></span></button>
-              </div>
-            </div>          
-            </section>
-  
-          <!--About Section-->
-          <section id="about">
-            <h2>About</h2>
-            <p class="about-content">
-              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
-              labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod
-            </p>
-            <button type="button"><span>Get my CV</span><span><img class="arrow-download" src="images/ic_download.svg" alt="arrow download"></span></button>
-            <div id="Languages">
-                <h3>Languages</h3>
-              <ul>
-                  <li>JavaScript(ES5/ES6)</li>
-                  <li>Ruby</li>
-                  <li>HTML</li>
-                  <li>CSS</li>
-              </ul>
-            </div>
-  
-            <div id="Frameworks">
-                <h3>Frameworks</h3>
-              <ul>
-                  <li>Bootstrap</li>
-                  <li>Ruby on rails</li>
-                  <li>RSpec</li>
-                  <li>Capibara</li>
-                  <li>Selenium</li>
-              </ul>
-            </div>
-  
-          <div id="Skills">
-              <h3>Skills</h3>
-            <ul>
-                <li>Codekit</li>
-                <li>Codepen</li>
```

Comment:
- [x] Great work adding the desktop version to your project, but unfortunately it's not showing up two main sections of the project the `About me` & `Get Started` as you can see in the screenshots below, Kindly, fix this issue🙏🏼☺️.

I think you delete them in this working branch by mistake

**Current mobile menu**           |  **Original design**
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/80895497/149597171-fb9471d0-0ee0-4657-a828-e9ac78268d4e.PNG)  |  ![](https://user-images.githubusercontent.com/80895497/149597174-5e2d09c8-e754-440b-b8f6-be50e9dea616.PNG)